Makefile

```diff
@@ -333,6 +341,7 @@ MATH_SRC +=\
 	$(SOURCEDIR)/Math/GPUWatcher.cu \
 	$(SOURCEDIR)/Math/GPURNGHandle.cu \
 	$(SOURCEDIR)/Math/MatrixQuantizerGPU.cu \
+	$(SOURCEDIR)/Math/NcclComm.cpp \
```

Comment:
Also, please move this line up to to end of of two blocks up (insert after line 323, before ifdef SUPPORT_AVX2).
Otherwise, I'm seeing Linux build breaks for the non-CUDA compile.